docker/datahub-ingestion-base/Dockerfile

```diff
@@ -105,57 +89,41 @@ ENV REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
 ENV SSL_CERT_FILE="/etc/ssl/certs/ca-certificates.crt"
 
 
-FROM base-empty AS full-deps-prebuild
+FROM python-base AS ingestion-base-slim
 
 USER 0
-RUN apt-get update && apt-get install --no-install-recommends -y -qq \
-    build-essential \
-    maven \
+RUN apt-get update && apt-get install -y \
+    python3-ldap \
+    libldap2-dev \
+    libsasl2-dev \
+    libsasl2-modules \
+    libaio-dev \
+    libaio1t64 \
+    libsasl2-modules-gssapi-mit \
+    krb5-user \
+    krb5-config \
+    libkrb5-dev \
+    librdkafka-dev \
+    ldap-utils \
+    unixodbc \
+    libodbc2 \
     && rm -rf /var/lib/apt/lists/*
 USER datahub
 
-RUN uv pip install python-ldap==3.4.4
-RUN --mount=type=cache,target=$HOME/.cache/uv,uid=1000,gid=1000 \
-    --mount=type=bind,source=./docker/datahub-ingestion/pyspark_jars.sh,target=/pyspark_jars.sh \
-    uv pip install python-ldap==3.4.4 pyspark~=3.5.0 && \
-    /pyspark_jars.sh
-
-FROM base-empty AS base-slim
-# Nothing to do here.
-
-FROM base-slim AS base-full
+FROM ingestion-base-slim AS ingestion-base-full
 
 USER 0
 RUN apt-get update && apt-get install --no-install-recommends -y -qq \
     default-jre-headless \
     && rm -rf /var/lib/apt/lists/*
 
-RUN if [ $(arch) = "x86_64" ]; then \
-    mkdir /opt/oracle && \
-    cd /opt/oracle && \
-    wget --no-verbose -c https://download.oracle.com/otn_software/linux/instantclient/2115000/instantclient-basic-linux.x64-21.15.0.0.0dbru.zip && \
-    unzip instantclient-basic-linux.x64-21.15.0.0.0dbru.zip && \
-    rm instantclient-basic-linux.x64-21.15.0.0.0dbru.zip && \
-    sh -c "echo /opt/oracle/instantclient_21_15 > /etc/ld.so.conf.d/oracle-instantclient.conf" && \
-    ldconfig; \
-    else \
-    mkdir /opt/oracle && \
-    cd /opt/oracle && \
-    wget --no-verbose -c https://download.oracle.com/otn_software/linux/instantclient/1923000/instantclient-basic-linux.arm64-19.23.0.0.0dbru.zip && \
-    unzip instantclient-basic-linux.arm64-19.23.0.0.0dbru.zip && \
-    rm instantclient-basic-linux.arm64-19.23.0.0.0dbru.zip && \
-    sh -c "echo /opt/oracle/instantclient_19_23 > /etc/ld.so.conf.d/oracle-instantclient.conf" && \
-    ldconfig; \
-    fi;
+RUN --mount=type=bind,source=./docker/datahub-ingestion/oracle_instantclient.sh,target=/oracle_instantclient.sh \
```

Comment:
Doesn't Oracle need libodbc?

Reply:
yes - libodbc2 is installed in ingestion-base-slim, and the full oracle driver is installed in ingestion-base-full